dream-journal/index.html

```diff
@@ -2,9 +2,16 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
+    <link rel="icon" type="image/svg+xml" href="/dreamjournal.ico" />
```

Comment:
اسم فایل باید kebab-case باشه

```suggestion
    <link rel="icon" type="image/svg+xml" href="/dream-journal.ico" />
```